server/events/repo_whitelist_checker_test.go

```diff
@@ -147,6 +147,13 @@ func TestRepoWhitelistChecker_IsWhitelisted(t *testing.T) {
 			"github.com",
 			true,
 		},
+		{
+			"should match if wildcard is not last character",
+			"github.com/owner/*-repo",
+			"owner/prefix-repo",
+			"github.com",
+			true,
+		},
```

Comment:
Can you add tests for the wildcard at the beginning of the string too: `*-owner/repo`? That will the cover all the cases I think.

Reply:
Thanks!  I added the new test case and ran the test.  If you are okay let me know and I can squash & merge.

Reply:
added

Reply:
So I meant at the beginning of the whitelist:
```go
		{
			"should match if wildcard is at beginning",
			"*-owner/repo",
			"prefix-owner/repo",
			"github.com",
			true,
		},
```

Reply:
So i'm a bit confused since all of the other tests start with "github.com" as the second value in the struct (= whitelist) and I thought this was required in the whitelist as per the below statement on this doc:

https://www.runatlantis.io/docs/server-configuration.html#repo-whitelist 

``Format is {hostname}/{owner}/{repo}, ex. github.com/runatlantis/atlantis``

Reply:
It's not required. You can do `--repo-whitelist='*'` for example to match everything.